### PR TITLE
Logging

### LINKED ISSUE #17: Add logging
### Enhancement Proposal

We need logs of course!

### PR DESCRIPTION
**Pull request by dstathis**
## Issue
Fixes #17 

## Solution
Uses python logging module to set up logging for COS Alerter.

## Testing Instructions
Try COS Alerter and see that the logs look how you would expect.

## Release Notes
Added logging

### CODE REVIEW

File: cos_alerter/daemon.py [hunk 1]
```diff
@@ -4,53 +4,119 @@
...
+    waitress_logger = logging.getLogger("waitress")
+    cos_logger.propagate = False
+    waitress_logger.propagate = False
+    cos_logger.setLevel(level=log_level)
+    waitress_logger.setLevel(level=log_level)
+    handler = logging.StreamHandler()
+    formatter = logging.Formatter("%(asctime)s:%(levelname)s:%(name)s:%(message)s")
+    handler.setFormatter(formatter)
+    cos_logger.addHandler(handler)
+    waitress_logger.addHandler(handler)
```
Comment (rbarry82) [inline cos_alerter/daemon.py:84]: I would be nice to have this as a separate block. All of the logging stuff is global, so if it were handled elsewhere (even by importing a module which handled the setup at the top level), the logging would be usable without necessarily going through this entrypoint.

File: cos_alerter/daemon.py [hunk 2]
```diff
@@ -4,53 +4,119 @@
...
     Args:
         run_for: This argument is for testing purposes. If set, only run for "run_for" seconds.
+        argv: Can be used to override the cli args for testing purposes.
     """
+    args = parse_args(argv[1:])
+
+    if args.log_level:
+        log_level = LOG_LEVEL_CHOICES[args.log_level]
+    else:
+        log_level = LOG_LEVEL_CHOICES[config["log_level"]]
```
Comment (Abuelodelanada) [inline cos_alerter/daemon.py:73]: This `else` can be removed.

File: cos_alerter/daemon.py (hunk 1 above)
Comment (Abuelodelanada) [inline cos_alerter/daemon.py:84]: Can these lines be a separate function?

**Review by Abuelodelanada — APPROVED** [review]

File: cos_alerter/daemon.py (hunk 2 above)
Comment (dstathis) [reply cos_alerter/daemon.py:73]: No it can't. The point is that the cli arg overrides the config arg.

File: cos_alerter/daemon.py (hunk 1 above)
Comment (dstathis) [reply cos_alerter/daemon.py:84]: See comment from @rbarry82 above.

Comment (dstathis) [reply cos_alerter/daemon.py]: Done.

Comment (dstathis) [reply cos_alerter/daemon.py]: Done.

**Pull request merged**